Sources/ToastWindow.swift

```diff
@@ -156,15 +156,13 @@ open class ToastWindow: UIWindow {
 
   @objc func keyboardWillShow() {
     self.bringWindowToTop()
-    for subview in self.originalSubviews.allObjects {
-      guard let subview = subview as? UIView else { continue }
+    for subview in self.originalSubviews.allObjects as! [UIView] {
```

Comment:
I don't think it is good to use force casting. I'd like to avoid it when possible.

Reply:
But it makes sense here: it's a private variable, so we're 100% controlling its type.

Reply:
Anyway, I've re-introduced the guard and removed a call to bringWindowToTop.